### PR TITLE
Documentation improved: mobile access

### PR DESCRIPTION
**Pull request by math-GH**
- Added some words to the mobile access page that gives information how to access the FreshRSS instances.
- Added Windows to the list of operation systems of known apps.
- Replaced "OS X" with "MacOS" on the list of operation systems of known apps.

### CODE REVIEW

File: README.fr.md [hunk 1]
```diff
@@ -217,7 +217,7 @@ Voir le [dépôt dédié à ces extensions](https://github.com/FreshRSS/Extensio
 
 # APIs et applications natives
 
-FreshRSS supporte l’accès depuis des applications natives pour Linux, Android, iOS, et OS X, grâce à deux APIs distinctes :
+FreshRSS supporte l’accès depuis des applications natives pour Linux, Android, iOS, Windows et MacOS, grâce à deux APIs distinctes :
```
Comment (Frenzie) [inline README.fr.md]: Or maybe `macOS`? (To be clear, I don't care.)

Comment (math-GH) [reply README.fr.md]: Oh, you are right.

File: docs/en/users/06_Mobile_access.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
 
-# Enable the API in FreshRSS
+# Mobile Access
+
+You can access FreshRSS on mobile devices via browser and via mobile apps.
+
+
+## Access via Browser
+
+The user interface of FreshRSS is optimzed to visit it on small and large screens. It means that it fits the content on small mobile device screens too. Just visit your FreshRSS instance with a common used internet browser (f.e. Chrome, Safari, Firefox).
```
Comment (Frenzie) [inline docs/en/users/06_Mobile_access.md]:
```suggestion
The FreshRSS is user interface is optimized for both small and large screens. The content will fit nicely on small mobile device screens as well.
```

File: docs/en/users/06_Mobile_access.md [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+
+
+## Access via Mobile App
+
+FreshRSS supports access from mobile / native apps for Linux, Android, iOS, Windows and MacOS, via two distinct APIs: Google Reader API (best), and Fever API (limited features and less efficient).
```
Comment (Frenzie) [inline docs/en/users/06_Mobile_access.md]: macOS?

File: docs/en/users/06_Mobile_access.md [hunk 4]
```diff
@@ -1,6 +1,23 @@
...
+
+A list of known apps is available on [FreshRSS GitHub page](https://github.com/FreshRSS/FreshRSS#apis--native-apps).
```
Comment (Frenzie) [inline docs/en/users/06_Mobile_access.md]:
```suggestion
A list of known apps is available on the [FreshRSS GitHub page](https://github.com/FreshRSS/FreshRSS#apis--native-apps).
```

**Review by Frenzie — APPROVED** [review]
lgtm

File: docs/en/users/06_Mobile_access.md [hunk 5]
```diff
@@ -1,6 +1,23 @@
...
 
-# Enable the API in FreshRSS
+# Mobile Access
+
+You can access FreshRSS on mobile devices via browser and via mobile apps.
+
+
+## Access via Browser
+
+The FreshRSS's user interface is optimized for both small and large screens. The content will fit nicely on small mobile device screens as well.
```
Comment (Frenzie) [inline docs/en/users/06_Mobile_access.md]:
```suggestion
The FreshRSS user interface is optimized for both small and large screens. The content will fit nicely on small mobile device screens as well.
```

Comment (math-GH) [reply docs/en/users/06_Mobile_access.md]: I am not sure which one is correct. I hope you know it better than me ;)

**Pull request merged**